rpc/ethereum/namespaces/debug/api.go

```diff
@@ -140,7 +140,6 @@ func (a *API) TraceTransaction(hash common.Hash, config *evmtypes.TraceConfig) (
 
 	traceTxRequest := evmtypes.QueryTraceTxRequest{
 		Msg:          ethMessage,
-		TxIndex:      uint64(transaction.Index),
```

Comment:
Can you manually set it to `0` instead and add a comment explaining the value?

Reply:
We dont have the TxIndex property anymore on the `QueryTraceTxRequest` type. I removed it on the change and reserved the id for it not to be used anymore. TxIndex was not used anywhere in the code anymore so I removed it. The cleaner this endpoint it the better